src/ufoLib2/objects/layerSet.py

```diff
@@ -131,7 +130,7 @@ def read(cls, reader: UFOReader, lazy: bool = True) -> "LayerSet":
         assert defaultLayer is not None
 
         self = cls(
-            layers,  # type: ignore
+            layers=layers,  # type: ignore
```

Comment:
why the `# type: ignore` in here and not also in the `new` classmethod above? let's remove the comment if we can

Reply:
Made the typing more accurate.